docs/01-app/03-api-reference/05-config/01-next-config-js/turbopackPersistentCaching.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: turbopackPersistentCaching
+description: Learn how to enable Persistent Caching for Turbopack builds
+version: canary
+---
+
+## Usage
+
+Turbopack Persistent Caching enables Turbopack to share state across multiple `next dev` or `next build` commands. When enabled, Turbopack will save and restore data to the `.next` folder between builds, which can greatly speed up subsequent builds and dev sessions.
+
+```ts filename="next.config.ts" switcher
+import type { NextConfig } from 'next'
+
+const nextConfig: NextConfig = {
+  experimental: {
+    turbopackPersistentCaching: true,
+  },
+}
+
+export default nextConfig
+```
+
+```js filename="next.config.js" switcher
+/** @type {import('next').NextConfig} */
+const nextConfig = {
+  experimental: {
+    turbopackPersistentCaching: true,
+  },
+}
+
+module.exports = nextConfig
```

Comment:
Should this contain a warning that this really really isn't stable yet (though there's `experimental` already in there of course)?